website/src/theme/BlogSidebar/Desktop/index.d.ts

```diff
@@ -1,9 +0,0 @@
-/**
- * Copyright (c) Meta Platforms, Inc. and affiliates.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- */
-/// <reference types="react" />
-import type { Props } from "@theme/BlogSidebar/Desktop";
```

Comment:
For these swizzled components, could you import the `Props` types and use them for each of the exported components?